pkg/controller/plan/adapter/ova/validator.go

```diff
@@ -100,3 +100,8 @@ func (r *Validator) MaintenanceMode(vmRef ref.Ref) (ok bool, err error) {
 func (r *Validator) DirectStorage(vmRef ref.Ref) (bool, error) {
 	return true, nil
 }
+
+// NO-OP
+func (r *Validator) StaticIPs(vmRef ref.Ref) (bool, error) {
+	return true, nil
```

Comment:
I wonder if there's any indication of static IPs in the OVF, even though I doubt it, but that's something for another day

Reply:
i doubt it as well, maybe @bkhizgiy saw something in a OVF recently. as far i can remember, the option is to save MACs.

Reply:
@ahadas @liranr23 Haha, I checked that after reviewing this PR yesterday, was having the same question:)
So yes, we can have a static IP address in the OVF, but it's not part of the basic configuration. It should be added manually as a property by the user (alongside other available configurations). The MAC, on the other hand, can be specified directly in the network configurations (and we are fetching this data). So, given it's not a standard configuration, I'm not sure if we want (or can) to support it..